pkg/console/errors/custom_logo_errors.go

```diff
@@ -0,0 +1,31 @@
+package errors
+
+// a sync progressing error captures the idea that the operand is in an incomplete state
+// and needs to be reconciled.  It should be used when the sync loop should abort, but
+// it implies the idea of "progressing", not "failure".  A built-in error type can be
+// used and returned when a true failure is encountered.
```

Comment:
Until we eliminate our use of pseudo errors (see the sync error we already have), using a custom error for logo errors.  API errors abort the loop, Sync errors do not, Logo errors do not.